ops/terraform/env/mgmt/alarms.tf

```diff
@@ -106,3 +106,39 @@ resource "aws_cloudwatch_metric_alarm" "ec2_failing_instances" {
   alarm_actions = [aws_sns_topic.victor_ops_alert.arn]
   ok_actions    = [aws_sns_topic.victor_ops_ok.arn]
 }
+
+## BFD-3520
+resource "aws_cloudwatch_log_metric_filter" "ec2_init_fail_count" {
+  name           = local.init_fail_filter_name
+  pattern        = local.init_fail_pattern
+  log_group_name = local.log_groups.cloudinit_out
+
+  metric_transformation {
+    name          = local.init_fail_metric_name
+    namespace     = local.this_metric_namespace
+    value         = "1"
+    default_value = "0"
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "ec2_init_fail" {
+  alarm_name          = local.init_fail_alarm_name
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = "1"
+
+  metric_name = local.init_fail_filter_name
+  namespace   = local.this_metric_namespace
+  period      = 60
+
+  statistic = "Maximum"
+  threshold = "0"
+
+  alarm_actions = [aws_sns_topic.victor_ops_alert.arn]
+
+  actions_enabled = true
+  alarm_description = join("", [
+    "At least 1 (see Alarm value for exact number) EC2 instance is failing startup steps.\n",
+    "See ${local.ec2_failing_instances_runbook_url} for instructions on resolving this alert."
```

Comment:
The Alarm definition here looks good! However, I believe this Alarm description should link a _different_ runbook for this particular Alarm than the `ec2_failing_instances` Alarm. You could probably get away with creating a placeholder Wiki page for the runbook so that you have a URL to add to `mgmt.yaml` which could then be added in-place of the `local.ec2_failing_instances_runbook_url` here.

Reply:
At this point, a failure is a failure is a failure from my perspective.

If we need a separate runbook page in the WIKI, it seems that tying this alert to a new WIKI page should be part of the Jira ticket to create that WIKI, no?

Reply:
I do agree that failures should be treated equally, and that perhaps we have a landing page of sorts for this class of failure. But, I _also_ feel that we should make each Wiki Runbook distinct so that the on-call responding to these types of alerts don't need to do too much digging on a given page to figure out how to narrow down the issue. There is no possibility that the on-call hastily follows the incorrect instructions for a different class of failure and ends up confused why there seems to be no issue.

Unfortunately, the means by which we inform the on-call _how_ to discern the failing instances (wiki page in Alarm description, command in Alarm description, link to Log query in description, etc.) was not captured in the ticket's ACs. In the interest of not increasing the scope of this ticket further, perhaps we just replace the link to the existing runbook with an obvious placeholder? We can then create a follow-up Story to write up the Wiki Runbook on how to determine what instances are failing and how to diagnose the problem for next Sprint.